src/components/MovieTile/MovieTile.vue

```diff
@@ -1,57 +1,59 @@
 <template>
-  <div class="movie-cell" v-on:click="handleMovieSelection">
-    <img :src="getPic(imageSrc)" class="poster" :alt="title" />
-    <p>
+  <div class="movie-tile" v-on:click="handleMovieSelection">
+    <img :src="imageSrc" class="poster" :alt="title" />
+    <p class="movie-tile__header">
       <span class="title">{{ title }}</span>
       <span class="year">{{ year }}</span>
     </p>
     <p class="genre">
-      {{ genre }}
+      <span v-for="(genre, index) in genres" :key="genre">{{ genre }}<span v-if="index < genres.length - 1">, </span></span>
```

Comment:
I liked previous implementation better, you can move all this logic into script section and it would be much cleaner 
```
<p class="genre">{{genresStr}}</p>

computed: {
 genresStr() {
    return this.genres.join(', ');
 }
}
```